content/en.json

```
@@ -92,13 +88,13 @@
       "subtitle": "We work in a modern way, aiming for quality. \"First time right\" is also cheapest in the end.",
       "how-title": "How we like to do this:",
       "how": [
-        "Modern version control",
-        "Up to date tooling, frameworks, and libraries",
+        "Everything in version control",
+        "Up-to-date tools",
         "Automation where possible",
         "Strong test coverage",
-        "Linting, code quality checks",
+        "Linting",
```

This looses information when going more brief I think? E.g. "Sonarqube" is listed as a concrete example further down, which does follow from "code quality checks" but not from just "linting".

Perhaps "Linting" _is_ included in code quality, so we could turn it around for brevity?

```suggestion
        "Code quality controls",
```